spring-boot-project/spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/web/reactive/webclient/ExampleId.java

```diff
@@ -18,10 +18,8 @@
 
 import java.util.UUID;
 
-import org.springframework.core.convert.converter.Converter;
-
 /**
- * An example attribute that requires a {@link Converter}.
+ * An example attribute that requires a {@link org.springframework.core.convert.converter.Converter}.
```

Comment:
Can you expand why you think this is a good idea?

Reply:
@snicoll I'm not sure this is generally acceptable or just my preference but the rationale behind the change is that when an imported class is referenced only in Javadoc, to prevent it from cluttering `import` section, I usually prefer to use FQN in Javadoc instead of `import`.

Reply:
That sounds totally reasonable but that's not what we do here.

Reply:
@snicoll Sorry but I don't get it. Could you give me some more pointer for the last comment?

Reply:
@snicoll I guess you meant you want to revert the change, right?

Reply:
I do. Here [is an example that we use quite a lot](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cache/CacheAutoConfiguration.java#L56).

Having said that, I've seen that is not consistent, see [AopAutoConfiguration](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/aop/AopAutoConfiguration.java#L29) for instance. I suggest to revert for now and I'll raise that at the next team call to check with the rest of the team.